src/index.js

```diff
@@ -18,6 +18,27 @@ const fileExists = filename => {
   }
 };
 
+const convertNodeModuleImport = stylusContent => {
+
+  const match = stylusContent.match(/~.*/);
+  const module = resolve('./node_modules', match[0]);
```

Comment:
`./` in this context means relative to the current directory, which I think is what we want, but let me know if that creates any issues.